include/maddy/strongparser.h

```diff
@@ -40,6 +40,23 @@ class StrongParser : public LineParser
    */
   void Parse(std::string& line) override
   {
+    // This version of the regex is changed exactly the same way
+    // that the regex for the emphasized parser was changed, and
+    // it then passes all the 'disabled' tests in the 'strong parser'
+    // test, but then it fails general parsing.  For some reason,
+    // "__text__" translates "<i></i>text<i></i>" even though there
+    // are no word boundaries at the correct places.  It's weird!
```

Comment:
The strong parser is usually handled before the emphasized one: https://github.com/progsource/maddy/blob/master/include/maddy/parser.h#L195
This is so, that double `_` or `*` can be easier determined as strong. Therefor I guess, that is why you only get italic tags, if you do not first run it through the strong parser.